packages/plugins/src/plugins/aave/helpers.ts

```diff
@@ -338,3 +355,99 @@ export function getElementLendingData(
   };
   return elementData;
 }
+
+export function getRates(
+  tokenAddress: string,
+  platformId: string,
+  formattedReserve: Pick<
+    FormattedReserve,
+    'supplyAPR' | 'variableBorrowAPR' | 'totalLiquidity' | 'totalDebt'
+  >,
+  elementName: string
+): BorrowLendRate {
+  const lendingApr = Number(formattedReserve.supplyAPR);
+  const borrowingApr = Number(formattedReserve.variableBorrowAPR);
+  const depositedAmount = Number(formattedReserve.totalLiquidity);
+  const borrowedAmount = Number(formattedReserve.totalDebt);
+  return {
+    tokenAddress,
+    borrowYield: {
+      apy: aprToApy(borrowingApr),
+      apr: borrowingApr,
+    },
+    borrowedAmount,
+    depositYield: {
+      apy: aprToApy(lendingApr),
+      apr: lendingApr,
+    },
+    depositedAmount,
+    platformId,
+    poolName: elementName,
+  };
+}
+
+export function getTokenPriceSourceFromReserve(
+  aTokenAddress: string,
+  networkId: NetworkIdType,
+  platformId: string,
+  formattedReserve: Pick<FormattedReserve, 'decimals' | 'underlyingAsset'>,
+  underlyingAssetPrice: TokenPrice,
+  elementName: string
+) {
+  const source: TokenPriceSource = {
+    id: platformId,
+    weight: 1,
+    address: aTokenAddress,
```

Comment:
```suggestion
    address,
```

PS: only if the above suggestion is accepted